solr/solrj/src/test/org/apache/solr/client/solrj/request/json/DirectJsonQueryRequestFacetingEmbeddedTest.java

```diff
@@ -38,7 +39,6 @@
 @SuppressSSL
 public class DirectJsonQueryRequestFacetingEmbeddedTest extends EmbeddedSolrServerTestBase {
 
-  private static final String COLLECTION_NAME = "techproducts";
```

Comment:
Why not leave this but merely change the value to collection1?  This way you avoid changing many lines below.

Reply:
Sure that's logical